src/components/EventDashboard/CCPTabPanel.tsx

```diff
@@ -108,16 +110,34 @@ const CCPTabPanel = ({ eventId }: { eventId: string }) => {
   const [orderBy, setOrderBy] = React.useState<string>('name');
   const classes = useStyles();
   const [order, setOrder] = React.useState<Order>('asc');
+  const [anchorEl, setAnchorEl] = React.useState<
+    null | (EventTarget & HTMLButtonElement)
+  >(null);
   const { data } = useQuery(GET_CCPS_BY_EVENT_ID, { variables: { eventId } });
 
   const rows = data ? data.collectionPointsByEvent : [];
+  const open = Boolean(anchorEl);
+  const history = useHistory();
+  const id = open ? 'simple-popper' : undefined;
```

Comment:
what does this do? shouldn't each popper have a unique id? also should any id ever be `undefined`?

Reply:
It was in the documentation and it must have been copied over, I'll remove it as I don't think the poppers need it.